local_ads/statistics.hpp

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include "local_ads/event.hpp"
+
+#include "base/thread.hpp"
+
+#include <chrono>
+#include <list>
+#include <map>
+#include <mutex>
+#include <string>
+#include <vector>
+
+namespace local_ads
+{
+class Statistics final
+{
+public:
+  struct PackedData
+  {
+    uint8_t m_eventType = 0;
```

Comment:
Does the order important? Is it possible to sort fields from the largest to smallest?